.travis.yml

```diff
@@ -7,3 +7,5 @@ notifications:
       - joeheth@gmail.com
     on_success: never
     on_failure: always
+before_install:
+  - gem install bundler
```

Comment:
Build on Travis CI did not succeed, due to:

"In short, the bundler version that Travis uses is very outdated. Running a gem update and install for bundler as a prestep for the Travis build fixes it usually."

Please refer to the issues described here: 
- https://github.com/rubygems/rubygems/issues/1419
- https://github.com/thoughtbot/factory_girl/pull/846
- https://github.com/travis-ci/travis-ci/issues/3531